functions/test/endpoints/FirestoreDataFactory.ts

```diff
@@ -265,6 +270,18 @@ export class FirestoreDataFactory{
         })
     }
 
+
+    static async getCompetitionPointsStatus(db: firebase.firestore.Firestore, house:string, user_id:string): Promise<CompetitionPointStatus> {
+        const user_doc = await db.collection("Users").doc(user_id).get()
+        const house_doc = await db.collection("House").doc(house).get()
+        const status: PointStatus = {
+            user_points: user_doc.data()!["TotalPoints"],
+            user_semester_points: user_doc.data()!["SemesterPoints"],
+            house_points: house_doc.data()!["TotalPoints"]
+        }
+        return new CompetitionPointStatus(status)
+    }
+
```

Comment:
This is a new helper function which will retrieve the current status of points for a user and their house. This should be used whenever you need to verify an endpoint modifies the points.